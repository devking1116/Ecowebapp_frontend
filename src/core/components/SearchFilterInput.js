/* @flow */
import React from 'react';
import { Select } from 'semantic-ui-react';

type Props = {
};

/**
 * The input for selecting search filters.
 */
export default function SearchFilterInput({ ...otherProps }: Props): React.Element<*> {
  return (
    <Select multiple {...otherProps} />
  );
}
