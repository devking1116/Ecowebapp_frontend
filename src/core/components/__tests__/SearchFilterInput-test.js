import React from 'react';
import renderer from 'react-test-renderer';

import SearchFilterInput from '../SearchFilterInput';

test('it displays the semantic-ui-react Select for search filters.', () => {
  const component = renderer.create(
    <SearchFilterInput options={[]} />
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
