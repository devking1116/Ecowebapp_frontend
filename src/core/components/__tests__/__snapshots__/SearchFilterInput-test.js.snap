exports[`test it displays the semantic-ui-react Select for search filters. 1`] = `
<div
  aria-busy={undefined}
  aria-disabled={undefined}
  aria-expanded={false}
  aria-multiselectable={true}
  className="ui multiple selection dropdown"
  onBlur={[Function]}
  onChange={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onMouseDown={[Function]}
  role="listbox"
  tabIndex={0}>
  <select
    aria-hidden="true"
    multiple={true}
    name={undefined}
    type="hidden"
    value={Array []}>
    <option
      value="" />
  </select>
  <div
    className="text" />
  <i
    aria-hidden="true"
    className="dropdown icon" />
  <div
    className="menu transition" />
</div>
`;
